test/common/fam_test_config.h.in

```diff
@@ -51,7 +51,7 @@ using namespace openfam;
 #cmakedefine TEST_FAM_CONTEXT_MODEL "${TEST_FAM_CONTEXT_MODEL}"
 #cmakedefine TEST_RUNTIME "${TEST_RUNTIME}"
 #cmakedefine TEST_NUM_CONSUMER "${TEST_NUM_CONSUMER}"
-
+#cmakedefine USE_CONFIG_OPTIONS "${USE_CONFIG_OPTIONS}"
 #ifdef MEMSERVER_PROFILE
 #cmakedefine TEST_MEMSERVER_IP "0:127.0.0.1"
 #endif
```

Comment:
Does line 55-57 (TEST_MEMSERVER_IP) also require change if USE_CONFIG_OPTIONS is set?